contribs/gmf/src/backgroundlayerselector/component.js

```diff
@@ -61,11 +61,11 @@ function gmfBackgroundlayerselectorTemplateUrl($element, $attrs, gmfBackgroundla
  *
  * Used metadata:
  *
- *  * thumbnail: The URL used for the icon.
+ *  * `thumbnail`: The URL used for the icon.
  *
  * Used functionnalities:
```

Comment:
functionalities
(same comment for occurrences in other files)

Reply:
Thanks, fixed :-)